include/aspect/utilities.h

```diff
@@ -280,7 +280,8 @@ namespace aspect
            */
           void set_points(const std::vector<double> &x,
                           const std::vector<double> &y,
-                          bool cubic_spline = true);
+                          bool cubic_spline = true,
+                          bool monotone_spline = false);
```

Comment:
Can you document this additional argument? Also maybe in the class documentation?

It's not clear to me whether you can have both cubic and monotone splines. Is this possible?